.obs/dockerfile/slem4r-base-iso/Dockerfile

```diff
@@ -1,13 +1,15 @@
 # SPDX-License-Identifier: Apache-2.0
-#!BuildTag: suse/sle-micro-rancher-iso/base-%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:%VERSION%
+#!BuildTag: suse/sle-micro-iso/base-%%SLEMICRO_VERSION%%:%VERSION%-%RELEASE%
```

Comment:
We should keep the 3 tags approach, this will be hand if we ever need to pin specific builds for specific CVE fixes or similar.